src/test/java/seedu/duke/commands/BalanceCommandTest.java

```diff
@@ -89,37 +89,36 @@ public void getBalance_ifCurrencyIsNotSpecified_shouldReturnAllCurrencies() {
                     "ALL",
                     accounts
             );
-            Account CNYAccount = null;
-            Account EURAccount = null;
-            for(Account account : output){
-                if(account.getCurrencyType().equals(Currency.EUR)){
-                    EURAccount = account;
-                }
-                else if(account.getCurrencyType().equals(Currency.CNY)){
-                    CNYAccount = account;
+            Account cnyAccount = null;
+            Account eurAccount = null;
+            for (Account account : output) {
+                if (account.getCurrencyType().equals(Currency.EUR)) {
+                    eurAccount = account;
+                } else if (account.getCurrencyType().equals(Currency.CNY)) {
+                    cnyAccount = account;
```

Comment:
A bit weird here. Why run this in a loop? Why not just get the account directly and assert that it is equal to the currency?